PL_2_komentarze.py

```diff
@@ -0,0 +1,2722 @@
+
+def otworztoler(flaga):
+    spistoler = {
+        'P': 'toler-przewody',
+        'S': 'toler-stalowe',
+        'K': 'toler-kompozyt',
+        'C': 'toler-corroflon'
+        }
+    startfile('bin\\'+spistoler[flaga]+'.pdf')
+
+def otworznowywaz():
+    matka = Toplevel()
+
+    matka.title('Dodaj opis przewodu')
+    matka.iconbitmap(r'bin\tubes.ico')
+
+    def dodajwazdolisty():
+        global wersjajezykowa
+        if wersjajezykowa.get() == 'PL':
+            try:
+                wbn = xlrd.open_workbook('//Srv-dokumenty/Swiadectwomat/bin/spis.xls', formatting_info=True)
+                adresspisu = '//Srv-dokumenty/Swiadectwomat/bin/spis.xls'
+            except FileNotFoundError:
+                wbn = xlrd.open_workbook('bin/spis.xls', formatting_info=True)
+                messagebox.showwarning('Zapis lokalny!','Wąż dodany do listy lokalnej, zapasowej. Wprowadzone informacje o wężu NIE będą utrwalone w spisie sieciowym!')
+                adresspisu = 'bin/spis.xls'
+            wsn = wbn.sheet_by_name('lista')
+            wbn1 = copy(wbn)
+            wsn1 = wbn1.get_sheet(0)
+        elif wersjajezykowa.get() == 'ENG':
+            try:
+                wbn = xlrd.open_workbook('//Srv-dokumenty/Swiadectwomat/bin/spis_en.xls', formatting_info=True)
+                adresspisu = '//Srv-dokumenty/Swiadectwomat/bin/spis_en.xls'
+            except FileNotFoundError:
+                wbn = xlrd.open_workbook('bin/spis_en.xls', formatting_info=True)
+                messagebox.showwarning('Zapis lokalny!','Wąż dodany do listy lokalnej, zapasowej. Wprowadzone informacje o wężu NIE będą utrwalone w spisie sieciowym!')
+                adresspisu = 'bin/spis_en.xls'
+            wsn = wbn.sheet_by_name('lista')
+            wbn1 = copy(wbn)
+            wsn1 = wbn1.get_sheet(0)
+        liczbawierszy = 0
+        for _ in wsn.col(0):
+            liczbawierszy += 1
+        k1 = kolumnaindeks.get()
+        k2 = kolumnatypweza.get()
+        k3 = kolumnaopisweza.get()
+        k4 = kolumnawarstwazewnetrzna.get()
+        k5 = kolumnawzmocnienie.get()
+        k6 = kolumnawarstwawewnetrzna.get()
+        k7 = kolumnatempmaksymalna.get()
+        k8 = kolumnatempminimalna.get()
+        k9 = kolumnasrednica.get()
+        k10 = kolumnacisnienierobocze.get()
+        k11 = kolumnacisnienierozerwania.get()
+        k12 = kolumnapromienstatyczny.get()
+        k13 = kolumnapromiendynamiczny.get()
+        k14 = kolumnaflagatolerancji.get()
+        parametrydododania = [k1, k2, k3, k4, k5, k6, k7, k8, k9, k10, k11, k12, k13, k14]
+        for col_index in range(0,14):
+            wsn1.write(liczbawierszy, col_index, parametrydododania[col_index])
+        wbn1.save(adresspisu)
+        messagebox.showinfo("Dodano wąż do listy!", "Dodano wąż do listy!\n\nGratuluję sukcesu!")
+        ladujlistewezy(wersjajezykowa.get())
+        matka.destroy()
+
+
+    Label(matka, text='Indeks węża:').grid(padx=2, pady=2, sticky=W, row=0, column=0)
```

Comment:
Jeśli stąd do linii 580 mamy fragment rysujący okno z polami i guzikami -> osobna funkcja np. `def drawNewHoseWindow()`. Jeśli nie, to co to robi tak z ciekawości?